pkg/cmd/server/origin/controller.go

```diff
@@ -143,6 +148,14 @@ func (c *MasterConfig) NewOpenShiftControllerPreStartInitializers() (map[string]
 	// this matches the upstream name
 	ret["serviceaccount-token"] = saToken.RunController
 
+	hpaController := kubernetes.HorizontalPodAutoscalerControllerConfig{
```

Comment:
whoops, I didn't mean to stick this in pre-start -- that was I typo.  I've got a local copy that's moved to the OpenShift controllers initializer area shortly (or was there a different spot that you were thinking?).  It has to be the OpenShift area because we need an OpenShift client handle.